common/sgx/collateral.c

```diff
@@ -194,6 +194,32 @@ static oe_result_t _parse_sgx_extensions(
     return result;
 }
 
+static oe_result_t _hex_crl_to_byte(
```

Comment:
That's a naive way for conversion for now. Cannot find a OE function to do that.
@mingweishih Should we make it a standard method and put it elsewhere?

Reply:
If this is currently used here, I think it's fine.